wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,202 @@
+- `Ctrl` + `Shift` + `Enter`: Play current map with [Cinema](/wiki/Cinema) mod
+- `Ctrl` + `A`: Toggle Auto mod
+- `Ctrl` + `F3` or `Ctrl` + `+`: Increase osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `F4` or `Ctrl` + `-`: Decrease osu!mania speed (only in osu!mania mode)
+- `Ctrl` + `Backspace`: Delete last word (when using search)
+- `Up` or `Down`: Change selection
+- `Left` or `Right`: Change beatmap set
+- `Page Up` or `Page Down`: Change selection (large)
+- `Enter`: Make selection
+- `Shift` + `Left Arrow` or `Shift` + `Right`: Change group
+- `Shift` + `Enter`: Open/close group
+
+## Mods screen
+
+**Notes:**
+
+- The key references below assumes that none of the mods are selected.
+- Some mods may require the player to press the key twice.
+
+- `Q`: [Easy](/wiki/Easy_mod)
+- `W`: [No Fail](/wiki/No_Fail)
+- `E`: [Half Time](/wiki/Half_Time)
+
+---
+
+- `A`: [Hard Rock](/wiki/Hard_Rock)
+- `S`: [Sudden Death](/wiki/Sudden_Death)
+- `S`, then `S`: [Perfect](/wiki/Perfect)
+- `D`: [Double Time](/wiki/Double_Time)
+- `D`, then `D`: [Nightcore](/wiki/Nightcore)
+- `F`: [Hidden](/wiki/Hidden) (osu!standard, osu!taiko, and osu!catch)
+- `F`: [Fade In](/wiki/Fade_In) (osu!mania only)
+- `F`, then `F`: [Hidden](/wiki/Hidden) (osu!mania only)
+- `G`: [Flashlight](/wiki/Flashlight)
+
+---
+
+- `X`: [Relax](/wiki/Relax)
+- `C`: [Auto Pilot](/wiki/Auto_Pilot)
+- `V`: [Spun Out](/wiki/Spun_Out)
+- `B`: Auto
+- `B`, then `B`: Cinema
+
+---
+
+- `3`: [3K](/wiki/3K) (osu!mania only)
+- `4`: [4K](/wiki/4K) (osu!mania only)
+- `5`: [5K](/wiki/5K) (osu!mania only)
+- `6`: [6K](/wiki/6K) (osu!mania only)
+- `7`: [7K](/wiki/7K) (osu!mania only)
+- `8`: [8K](/wiki/8K) (osu!mania only)
+- `9`: [9K](/wiki/9K) (osu!mania only)
+
+---
+
+- `1`: Deselect selected mods
+- `2`: Closes the mod selection dialog
+
+## Play Mode
+
+- `+` or `-`: Adjust local offset (hold `Alt` to change the precision).
+- `Esc`: Pause or quit song if Auto or Cinema mod is used.
+- `Space`: Skip opening cut-scene.
+- `Tab`: Toggle scoreboard
+- `Shift` + `Tab`: Toggle in-game interface
+- `F3` or `F4`: Adjust speed for osu!mania.
+- `Ctrl` + `R` or `` ` `` (hold briefly): Quick retry the current map (autoskips the cut-scene upon retry)
+
+## Edit Mode
+
+### General
+
+- `Z`: Stop
+- `X`: Play from start
+- `C` or `Space`: Pause/Resume
+- `Left Arrow` or `Right Arrow`: Skip through the song.
+- `Shift` + `Left Arrow` or `Shift` + `Right Arrow`: Skip through the song at 4x speed.
+
+- `Ctrl` + `B`: Add bookmark at current location.
+- `Ctrl` + `Shift` + `B`: Remove bookmark at current location.
+- `Ctrl` + `Left Arrow`: Jump to previous bookmark.
+- `Ctrl` + `Right Arrow`: Jump to next bookmark.
+
+- `Ctrl` + `L`: Revert to saved
+- `Ctrl` + `S`: Quick save
+- `Ctrl` + `N`: Remove all notes (but keep timing/song setup)
+
+- `F1`: Compose mode
+- `F2`: Design mode
+- `F3`: Timing mode
+- `F4`: Song setup
 
 #### Compose mode
```

Comment:
`Composer`? We should use what it's called in the interface article tbh, too lazy to go over tho 🙃